### ISSUE TITLE
Packaged SteamControl.exe does not update steamcontrol.xml

### ISSUE THREAD

**Issue by jawadmn** [hadokenxgaming/hxg-obs-scoreboards#1]
Furthermore, it seems to be a different version (no flag options and such, GUI is pretty different) from the one in your video. Using blue-black-solid-scoreboard package, and have it successfully running otherwise (debugged for a while there to figure out exactly what was happening). Editing xml manually at the moment

**Comment by hadokenxgaming** [comment]
Yes, based on the .xml file you use will change the available fields given in the steamcontrol.exe. So sorry as I just now saw this. Were you able to work around the issue?
